app/controllers/api/groups_controller.rb

```diff
@@ -56,4 +56,8 @@ def accessible_records
     Queries::ExploreGroups.new
   end
 
+  def resource_options(scope:, serializer:, root:)
```

Comment:
Done!

On Wed, May 25, 2016 at 1:12 PM, Robert Guthrie <notifications@github.com>
wrote:

> In app/controllers/api/groups_controller.rb
> <https://github.com/loomio/loomio/pull/3390#discussion_r64644562>:
>
> > @@ -56,4 +56,8 @@ def accessible_records
> >      Queries::ExploreGroups.new
> >    end
> >
> > +  def resource_options(scope:, serializer:, root:)
>
> cool. maybe leave a comment to highlight if/when someone is expecting same
> behaviour?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3390/files/97c938e5d5bc135865a831c49871e441cad6ebc2#r64644562>
>